README.md

```diff
@@ -23,7 +23,9 @@ Because theming is achieved by generating multiple classes, **legacy browsers ar
 ---
 
 - [Requirements](#requirements)
-- [tl;dr (React version)](#tldr-react-version)
+- [Example Usage](#example-usage)
+  - [Basic Usage](#basic-usage)
+  - [Themed Usage (React version)](#themed-usage-react-version)
```

Comment:
Bit weird that we only mention React in the theme stuff but the basic usage is React as well. Not sure if that's confusing.

Reply:
I'd be keen to use a non-React demo for the basic usage example. Hard to make it readable, though. Any ideas?

Reply:
Maybe ejs or something? Feels kinda dated though.

Reply:
Could just make the example create an HTML template literal, not an entire component.